reference/configuration/tidb-server/configuration-file.md

```diff
@@ -111,6 +111,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：[]
 + 默认情况下，该 list 名单为空，表示没有所需修复的坏表信息。
 
+### `max-server-connections`
+
++ TiDB 中同时允许的最大客户端连接数，用于资源控制。
++ 默认值：0
++ 默认情况下，TiDB 不限制客户端连接数。当本配置项的值大于 `0` 且客户端连接数到达此值时，TiDB 服务端将会拒绝新的客户端连接。
+
+### `max-index-length`
+
++ 用于设置新建索引的长度限制。
```

Comment:
should put the difference with MySQL's link here?

Reply:
+1, we can add a little more explainations here.

Reply:
MySQL does not have this item.

Reply:
@bb7133 @AilinKid PTAL

Reply:
nice work